### PR TITLE
Refactoring values

### PR DESCRIPTION
**Pull request by sarakoteli**

### CODE REVIEW

**Comment by sarakoteli** [comment]
I want to mention that these changes are only for better looking at this moment, because when I try to remove the part under psql: and postgresql: it causes an error where it asks for the password value, so this means that it does not take from the value defined above.
` Error: UPGRADE FAILED: error validating "": error validating data: [unknown object type "nil" in Secret.stringData.postgresql-password, unknown object type "nil" in Secret.stringData.postgresql-postgres-password]
`
Also for the DatastoreRODBPass that we discuss earlier, should I leave it as it is or modify it like we talked?
@Filip3mac

File: values.yaml [hunk 1]
```diff
@@ -358,10 +344,8 @@ solr:
     configsetName: ckanConfigSet
 
 
-
 postgresql:
-  # postgresql.enabled -- Flag to control whether to deploy PostgreSQL
-  enabled: true
```
Comment (Filip3mac) [inline values.yaml:364]: This sould be set to true on default so we can revert this.

File: values.yaml [hunk 2]
```diff
@@ -313,38 +309,28 @@ solr:
...
-  # solr.collection -- the name of the collection created by solr 
-  # since we are creating one with solr-init this needs to be blank
   collection:
-  # solr.collectionShards -- Number of shards for the SOLR collection
   collectionShards: 
-  # solr.collectionReplicas -- Number of replicas for each SOLR shard
   collectionReplicas: 
-  # solr.fullnameOverride -- Name override for the SOLR deployment
   fullnameOverride: *SolrName
-  # solr.replicaCount -- Number of SOLR instances in the cluster
```
Comment (Filip3mac) [inline values.yaml:331]: These comments should be here unless this is an error with github

File: values.yaml [hunk 3]
```diff
@@ -180,6 +149,34 @@ ckan:
...
+    RwDbPassword: *DatastoreRWDBPass
+    # ckan.datastore.RoDbUrl -- Url of the PostgreSQL server where the datastore database is hosted
+    RoDbUrl: *DBHost
+    # ckan.datastore.RoDbName -- Name of the database to be used for Datastore
+    RoDbName: *DatastoreDBName
+    # ckan.datastore.RoDbUser -- Username for the datastore database with read permissions
+    RoDbUser: *DatastoreRODBUser
+    # ckan.datastore.RoDbPassword -- Password for the datastore read permissions user
+    RoDbPassword: *DatastoreRODBPass
+
```
Comment (Filip3mac) [inline values.yaml:179]: Does this have proper indentation?

**Comment by Filip3mac** [comment]
> I want to mention that these changes are only for better looking at this moment, because when I try to remove the part under **psql:** and **postgresql:** it causes an error where it asks for the password value, so this means that it does not take from the value defined above. `Error: UPGRADE FAILED: error validating "": error validating data: [unknown object type "nil" in Secret.stringData.postgresql-password, unknown object type "nil" in Secret.stringData.postgresql-postgres-password]` Also for the DatastoreRODBPass that we discuss earlier, should I leave it as it is or modify it like we talked? @Filip3mac

Leave it for now.

File: values.yaml (hunk 1 above)
Comment (sarakoteli) [reply values.yaml:364]: Yes, I know it looks like it's left over from the tests, I'll change it right away.

File: values.yaml (hunk 3 above)
Comment (sarakoteli) [reply values.yaml:179]: They need to be under ckan: , right?

File: values.yaml [hunk 4]
```diff
@@ -331,20 +328,19 @@ solr:
   # solr.replicaCount -- Number of SOLR instances in the cluster
   replicaCount: 1
   volumeClaimTemplates:
-    # solr.volumeClaimTemplates.storageSize -- Size of Solr PVC
+    # solr.volumeClaimTemplates.storageSize -- Size of Solr PV
```
Comment (Filip3mac) [inline values.yaml:331]: @sarakoteli Why is this changed from pvc to pv?

File: values.yaml (hunk 3 above)
Comment (Filip3mac) [reply values.yaml:180]: Yes

File: values.yaml [hunk 5]
```diff
@@ -210,7 +208,6 @@ securityContext: {}
   # runAsUser: 1000
 
 service:
-  # service.type -- Type of the service created for the CKAN pod
```
Comment (Filip3mac) [inline values.yaml:213]: This comment should exist

File: values.yaml [hunk 6]
```diff
@@ -152,15 +128,8 @@ ckan:
   issues:
     sendEmailNotifications: "true"
 
-  # ckan.extraEnv -- An array to add extra environment variables
-  # For example:
-  # extraEnv:
-  #   - name: FOO
-  #     value: "bar"
-  #
```
Comment (Filip3mac) [inline values.yaml:160]: This comment should exist

File: values.yaml [hunk 7]
```diff
@@ -93,48 +93,24 @@ ckan:
   uwsg_num: "2"
   # ckan.maintenanceMode -- Set to true to disable CKAN from starting and serve a maintenance page
   maintenanceMode: "false"
+
   psql:
-    # ckan.psql.runOnAzure -- Set to true to run on Azure (if true wont run on anything other then azure) set to false to run on other platforms
     runOnAzure: false
-    # ckan.psql.initialize -- Flag whether to initialize the PostgreSQL instance with the provided users and databases
```
Comment (Filip3mac) [inline values.yaml:99]: This comment should exist

File: values.yaml (hunk 3 above)
Comment (sarakoteli) [reply values.yaml:180]: So they are

**Comment by Filip3mac** [comment]
Is this commit [9dee813](https://github.com/keitaroinc/ckan-helm/pull/113/commits/9dee813a8ad33bc863eb79e446b1f89aa18922f1) tested? @sarakoteli

**Comment by sarakoteli** [comment]
> Is this commit [9dee813](https://github.com/keitaroinc/ckan-helm/pull/113/commits/9dee813a8ad33bc863eb79e446b1f89aa18922f1) tested? @sarakoteli

Yes, it has been tested and passed without any issue.

**Review by Filip3mac — APPROVED** [review]
